tests/routers/test_p2p.py

```
@@ -72,7 +72,7 @@ def test_create_network_without_psk_auto_generates_psk(authenticated_client):
 
 
 def test_join_peer_creates_record(authenticated_client, monkeypatch):
-    def _sync_peer_manifest(db, *, peer, psk_hash, local_host=None, local_port=None):
+    def _sync_peer_manifest(db, *, peer, psk_hash, **kwargs):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_sync_peer_manifest`

Add return type annotation: `None`

(ANN202)